library/src/main/java/com/phillipcalvin/iconbutton/IconButton.java

```diff
@@ -18,6 +18,7 @@
 
     private enum DrawablePositions {
         NONE,
+        BOTH,
```

Comment:
yeah definitely makes sense. i thought `LEFT_AND_RIGHT` is a tad bit more succinct?